app/src/test/java/org/simple/clinic/drugs/selection/entry/CustomPrescriptionEntryControllerTest.kt

```diff
@@ -263,19 +262,23 @@ class CustomPrescriptionEntryControllerTest {
     whenever(prescriptionRepository.prescriptionImmediate(prescriptionUuid)).thenReturn(Just(prescription))
 
     //when
-    setupController(OpenAs.Update(prescriptionUuid))
+    createSheetAsUpdate(prescriptionUuid)
 
     //then
     verify(uiActions).finish()
   }
 
-  private fun setupController(openAs: OpenAs) {
-    controller = CustomPrescriptionEntryController(prescriptionRepository, openAs)
+  private fun createSheetAsNew(patientUuid: UUID) {
+    val openAsNew = OpenAs.New(patientUuid)
+    instantiateFixture(openAsNew)
+  }
 
-    uiEvents
-        .compose(controller)
-        .subscribe { uiChange -> uiChange(ui) }
+  private fun createSheetAsUpdate(prescriptionUuid: UUID) {
+    val openAsUpdate = OpenAs.Update(prescriptionUuid)
+    instantiateFixture(openAsUpdate)
+  }
```

Comment:
Nice! Can we rename these methods to be more use-case focused?

Ex: `createSheetForRecordingNewPrescription` and `createSheetForUpdatingPrescription`.